src/prop_current/cbsd_antenna_gain.py

```diff
@@ -0,0 +1,24 @@
+'''
```

Comment:
can you add this doctring at the top of the file
https://github.com/Wireless-Innovation-Forum/Spectrum-Access-System/blob/master/src/harness/sas.py#L1